nifi-nar-bundles/nifi-mongodb-bundle/nifi-mongodb-processors/src/main/java/org/apache/nifi/processors/mongodb/PutMongo.java

```diff
@@ -159,6 +159,7 @@ public List<PropertyDescriptor> getSupportedPropertyDescriptors() {
     @Override
     protected Collection<ValidationResult> customValidate(final ValidationContext validationContext) {
         List<ValidationResult> problems = new ArrayList<>();
+        problems.addAll(super.customValidate(validationContext));
```

Comment:
Was this an oversight earlier, to not call the super version?

Reply:
Yes.